docs/style.md

```diff
@@ -0,0 +1,28 @@
+# Style Guide
+
+## Generic
+
+* This guide can be ignored for data files that are not to be viewed in an editor.
+* 120 character per line for all source files.
+* Avoid white space errors like trailing spaces.
+
+
+## Python Code
+
+* We mostly follow [PEP8](https://peps.python.org/pep-0008/). Read below to see the differences.
+* 120 characters per line. PyCharm does this automatically, other editors can be configured for it.
+* Strings in core code will be `"strings"`. In other words: double quote your strings.
+* Strings in worlds should use double quotes as well, but imported code may differ.
+* Prefer [format string literals](https://peps.python.org/pep-0498/) over string concatenation,
+  use single quotes inside them: `f"Like {dct['key']}"`
+* Use type annotation where possible.
+
+
+## Markdown
+
+* We almost follow [Google's styleguide](https://google.github.io/styleguide/docguide/style.html).
+  Read below for differences.
+* For existing documents, try to follow its style or ask to completely reformat it.
+* 120 characters per line.
+* One space between bullet/number and text.
+* No lazy numbering.
```

Comment:
Why not? this is a feature that makes editing easyer.

Reply:
Couple of reasons:
1. Lazy numbering is harder to read in source form unless you use more space(s) than we do
1. which means harder to modify/correlate to the render
1. Lazy numbering shines where a list gets longer than what is probably good
1. Some text editors will automatically change the numbers - i.e. typing enter in github will put `2.` there even if there have been multiple (lazy) `1.` before and/or after
1. Some text editors allow renumbering or even do that automatically (PyCharm), so that would break consistency since we are all mostly using PyCharm.